wiiu/vino/vino.ts

```diff
@@ -209,29 +296,29 @@ export default interface VinoAPI {
 	navi_setBaseVisibility: (unk: boolean) => void;
 
 	/**
-	 * Unknown
-	 * @param unk - Unknown
+	 * Enables a light blue background on the navigation box
+	 * @param light - Enable light blue background if true
 	 */
-	navi_setBaseVisibilityOnKeyEvent: (unk: boolean) => void;
+	navi_setBaseVisibilityOnKeyEvent: (light: boolean) => void;
 
 	/**
-	 * Unknown
+	 * Resets any existing navigation box
 	 */
 	navi_reset: () => void;
 
 	/**
-	 * Unknown
+	 * Resets any existing navigation box, same as vino.navi_reset()?
 	 */
 	navi_resetWithoutDOMForcus: () => void;
 
 	/**
-	 * Unknown
+	 * Resets any navigation box with an selecting animation
```

Comment:
This description doesn't make much sense, I have no idea what this is supposed to mean?